### DISCUSSION TITLE
Odd character in firewall list

### DISCUSSION THREAD

**Discussion by FritzLott** [Security-Onion-Solutions/securityonion#5707] [Unsupported Versions] [answered]
Here is my terminal session:

[admin@securityonion ~]$ sudo so-firewall includedhosts analyst
127.0.0.1

192.168.168.150

There's some odd character between 127.0.0.1 and 192.168.168.150.  How can I delete that odd character and its odd IP address?  Cutting and pasting does NOT seem to work.

**Comment by m0duspwnens** [comment, marked as answer]
@FritzLott Open the file located at `/opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml` with vi, and check for special characters. If you are unable to see it, you may need to type `:set list` to show special characters within vi.

**Reply by FritzLott** [reply]
I used sudo vim /opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml

The odd character was "\e" and was easily deleted.

Thank you for your kind and prompt reply.
